inst/rmarkdown/templates/trb/resources/template.tex

```diff
@@ -0,0 +1,80 @@
+% Repository:  https://github.com/chiehrosswang/TRB_LaTeX_tex
+%
+% Transportation Research Board conference paper template
+% version 4.0 Lite (updates made to be compatible in Overleaf and ShareLaTeX)
+%
+%
+% When numbered option is activated, lines are numbered.
+\documentclass[numbered]{trbarticle}
+\usepackage{graphicx}
+\usepackage{booktabs}
+
+\newread\somefile
+\usepackage{xparse}
+
+\usepackage{natbib}
+\bibliographystyle{unsrtnat}
+\setcitestyle{round}
+
+% \usepackage[colorlinks=true,linkcolor=blue,citecolor=blue]{hyperref}
+% For TRB version hide links
+\usepackage[hidelinks]{hyperref}
+
+% Put here what will go to headers as author
+\AuthorHeaders{$runningheader$}
+\title{$title$}
+
+% TODO: add macros for easier formatting of \author.
+\author{%
+  $for(author)$
+  \textbf{$author.name$}\\
+  \textit{$if(author.note)$$author.note$\\$endif$}
+  $author.position$\\
+  $author.affiliation$\\
+  $author.email$\\
+  \hfill\break
+  $endfor$
+}
+
+% If necessary modify the number of words per table or figure default is set to
+% 250 words per table
+% \WordsPerTable{250}
+
+% If words are counted manually, put that number here. This does not include
+% figures and tables. This can also be used to avoid problems with texcount
+% program i.e. if one does not have it installed.
+\TotalWords{$wordcount$}
+
+\begin{document}
+\maketitle
+
+
+\section{Abstract}
+$abstract$
+\hfill\break%
+\hfill\break%
+\noindent\textit{Keywords}: $if(keywords)$ $for(keywords)$$keywords$, $endfor$ $endif$
+\newpage
+
+$body$
+
+\newpage
+$if(natbib)$
+$if(bibliography)$
+$if(biblio-title)$
+$if(book-class)$
+\renewcommand\bibname{$biblio-title$}
+$else$
+\renewcommand\refname{$biblio-title$}
+$endif$
+$endif$
+\bibliography{$bibliography$}
```

Comment:
As other templates in this package, in case there is several files
```suggestion
\bibliography{$for(bibliography)$$bibliography$$sep$,$endfor$}
```